### ISSUE TITLE
[Components] google-ads

### ISSUE THREAD

**Issue by pipedream-component-development** [PipedreamHQ/pipedream#9885]
# google_ads
## URLs
- https://developers.google.com/google-ads/api/docs/start
#### Prompt
Adds a contact to a specific customer list. Requires a prop of contact email. Lists typically update in 6 to 12 hours after operation.
#### URLs
- https://developers.google.com/google-ads/api/docs/remarketing/audience-segments/customer-match/get-started
- https://developers.google.com/google-ads/api/samples/add-customer-match-user-list

**Comment by sergio-eliot-rodriguez** [comment]
No, not for the Google accounts I've used for the project with Pipedream.

I once setup for another company as a freelancer and Google required to send a document with the indented use case and finally got the token, but I can't share it.

I don't think I tackled Google Ads for the project more likely @malexanderlim can get it from the assets obtained when it was onboarded to the app store.
